### PR TITLE
[zh]Resync content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md(reference-2)

### PR DESCRIPTION
**Pull request by howieyuen**
related: https://github.com/kubernetes/website/issues/34221

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md [hunk 1]
```diff
@@ -739,18 +695,60 @@ Path to kubeconfig file with authorization information (the master location is s
 </tr>
 
 <tr>
-<td colspan="2">--machine-id-file string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<!--Default:-->默认值："/etc/machine-id,/var/lib/dbus/machine-id"</td>
+<td colspan="2">--log_backtrace_at &lt;<!--a string in the form 'file:N'-->“file:N” 格式的字符串&gt;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<!--Default:-->默认值：:0</td>
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md:698]:
```suggestion
<td colspan="2">--log_backtrace_at &lt;<!--a string in the form 'file:N'-->“file:N” 格式的字符串&gt;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<!--Default:-->默认值：0</td>
```

File: content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md [hunk 2]
```diff
@@ -928,6 +936,36 @@ The previous version for which you want to show hidden metrics. Only the previou
 </td>
 </tr>
 
+<tr>
+<td colspan="2">--skip_headers</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
+<!-- If true, avoid header prefixes in the log messages -->
+如果为 true，请避免在日志消息中使用头部前缀
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md:945]:
```suggestion
如果为 true，則避免在日志消息中使用头部前缀
```

File: content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md [hunk 3]
```diff
@@ -54,92 +54,76 @@ kube-proxy [flags]
 <tbody>
 
 <tr>
-<td colspan="2">--azure-container-registry-config string</td>
+<td colspan="2">--add_dir_header</td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
+td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md:60]:
```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
```

File: content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md [hunk 4]
```diff
@@ -415,7 +372,6 @@ CSIMigrationPortworx=true|false (ALPHA - 默认值=false)<br/>
 CSIMigrationRBD=true|false (ALPHA - 默认值=false)<br/>
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md:372]: May add a `<br>` to line 352 could be better.

**Review by Sea-n — COMMENTED** [review]
Thanks!

Preview: https://deploy-preview-34242--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/command-line-tools-reference/kube-proxy/

File: content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md [hunk 5]
```diff
@@ -942,6 +980,16 @@ How long an idle UDP connection will be kept open (e.g. '250ms', '2s').  Must be
 </td>
 </tr>
 
+<tr>
+<td colspan="2">-v, --v int</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
+<!-- number for the log level verbosity -->
+设置 kubelet 日志级别详细程度的数值。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/kube-proxy.md:989]:
```suggestion
设置日志级别详细程度的数值。
```

**Pull request merged**